### PR TITLE
Feature: Dockerfile

### PR DESCRIPTION
**Pull request by Tandashi**
**What is included in this Pull Request:**
Docker support using a Dockerfile

**Why a Dockerfile?**
For people who don't want to install all the dependencies on Linux for example but already have docker installed.

**What the Dockerfile does:**
It downloads the latest build version and runs it

To download the latest version it uses the appveyor API and [jq](https://stedolan.github.io/jq/) to parse the json in the `sh` file.

**Future improvements:**
Host the docker image publicly so one doesn't have to build it themselves e.g. on [DockerHub](https://hub.docker.com)

### CODE REVIEW

File: Docker/start-latest.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (ORelio) [inline Docker/start-latest.sh:1]: This script is run with `bash` by the docker file, maybe use `/bin/bash` here for consistency or run with `sh`?

File: Docker/start-latest.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+cd /opt/data || exit 1
+
+# Download latest version
+curl -L https://ci.appveyor.com/api/buildjobs/51an4dvs204ak6tq/artifacts/MinecraftClient%2Fbin%2FRelease%2FMinecraftClient.exe --output MinecraftClient.exe
```
Comment (ORelio) [inline Docker/start-latest.sh:6]: That `51an4dvs204ak6tq` part is randomized by AppVeyor on every new build so getting the latest release would be slightly more involved (request https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts first to determine the latest download link)

**Comment by ORelio** [comment]
Hi @Tandashi, thanks for contributing. Adding a way to set up quickly with Docker is a good idea 👍

**Review by ORelio — CHANGES_REQUESTED** [review]
Here are a few remarks about your current code:

File: Docker/start-latest.sh (hunk 1 above)
Comment (Tandashi) [reply Docker/start-latest.sh:1]: Should be fixed. (957dc6e8f882517049be136094b12fd557143da5)

File: Docker/start-latest.sh (hunk 2 above)
Comment (Tandashi) [reply Docker/start-latest.sh]: Should be fixed. (957dc6e8f882517049be136094b12fd557143da5)

I use the appveyor API and parse the JSON using `jq` to get the latest Job ID  to curl the artifact.

**Review by ORelio — APPROVED** [review]

**Comment by ORelio** [comment]
Looks good now! Merged, thanks again 😉

**Pull request merged**